packages/core/src/auth/activation.ts

```diff
@@ -25,6 +41,12 @@ export async function initialize(loginManager: LoginManager): Promise<void> {
     await showManageConnectionsOnStartup()
 }
 
+function initializeCredentialsProviderManager() {
+    const manager = CredentialsProviderManager.getInstance()
+    manager.addProviderFactory(new SharedCredentialsProviderFactory())
+    manager.addProviders(new Ec2CredentialsProvider(), new EcsCredentialsProvider(), new EnvVarsCredentialsProvider())
+}
```

Comment:
This is a direct copy paste of this already existing function. Please export and re-use this existing function. You will need to move it to a shared file, e.g. `core/src/auth/utils.ts`
https://github.com/aws/aws-toolkit-vscode/blob/5e8ad316718661bed35230e3596c13e5e2b53f30/packages/core/src/extensionNode.ts#L313-L317

Reply:
will do